src/components/CRadioList/CRadioList.js

```diff
@@ -7,11 +7,11 @@ const getListeners = (context) => {
   const self = context;
 
   const listeners = {
-    input(payload) {
+    change(payload) {
       self.value = payload;
       const items = self.value;
       self.sendToEventBus('Changed', { items });
-      self.$emit('input', payload);
+      self.$emit('change', payload);
```

Comment:
Any special reason for emitting 'change' event instead of 'input'. We have used 'input' for each input change (no matter if Vuetify event was input or change) so far; that way Form also doesn't need two listeners inside `getFormInputs`.